### DISCUSSION TITLE
StreamLit

### DISCUSSION THREAD

**Discussion by Joseph-Kassab** [DrAlzahraniProjects/csusb_fall2024_cse6550_team2#4] [Coding]
**Professor's Instructions:**

All students must contribute to all 24 topics, including tasks such as testing, improving documentation, reviewing code, enhancing performance, creating guides, troubleshooting, and assisting with deployment and integration.

**Guidelines:**

- Leave comments to write your contributions every week in the corresponding task discussion.

- Do not copy and paste large content (more than 10 lines), except for relevant error messages.

- Links can be shared.

- For error diagnosis, input errors into ChatGPT and share the link to the ChatGPT response for suggestions.

**Comment by farheen-akhter-code** [comment]
Hi 
I added streamlit UI design for the app, please take the latest pull and let me know how it looks like, we can make changes to it as needed.

**Comment by Prasanth-Kandikatla** [comment]
Hi,
I updated rating icons for each response and tooltips for the icons, please take the latest pull and let me know if there are any issues.

**Comment by Arungodugu** [comment]
Hi,
I have updated the UI with animations and made the website more attractive

**Comment by VishnuTejaG07** [comment]
Hi,
I have updated the pending stat reports for the chatbot

**Comment by shreya192-edu** [comment]
Hi, 
I have added another stat report to the chatbot UI

**Comment by ketha-shishir** [comment]
Hi,
I have added the "Improvement over time" stat report

**Comment by VishnuTejaG07** [comment]
Hello again,
I have coordinated with Arun and updated the pending elements in UI

**Comment by ERIGELASANDHYA** [comment]
Hello,
added the final stat report in the sidemenu of the chatbot
